<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newRouteDlg</class>
 <widget class="QDialog" name="newRouteDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="src_cbx">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>起点：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>终点：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dest_cbx">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>途经城市数量：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cityNum_lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="passby_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>旅行策略：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="strategy_cbx">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>最少花费</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>最短用时</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>限制时间最少花费</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>计算路线</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
